configs/components/cpp-pcp-client.rb

```diff
@@ -69,6 +69,14 @@
     ["#{make} -j$(shell expr $(shell #{platform[:num_cores]}) + 1)"]
   end
 
+  # Can't run binaries when cross-compiling, and Solaris has an issue with
+  # accessing the certs while testing. AIX tests are always iffy.
+  if !platform.is_cross_compiled? && !platform.is_solaris? && !platform.is_aix?
```

Comment:
Every unit test suite has a SIGABRT somewhere on AIX. It'll require more digging to figure out why.